icons/envato.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Envato</title><path d="M20.058 1.043C16.744-2.841 6.018 4.682 6.104 14.38a.459.459 0 0 1-.45.451.459.459 0 0 1-.388-.221 10.387 10.387 0 0 1-.412-7.634.42.42 0 0 0-.712-.412 10.284 10.284 0 0 0-2.784 7.033A10.284 10.284 0 0 0 11.76 23.999c14.635-.332 11.257-19.491 8.298-22.956z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Envato</title><path d="M13.104 24c.593 0 1.074-.481 1.074-1.073 0-.593-.481-1.074-1.074-1.074-.592 0-1.073.481-1.073 1.074 0 .592.481 1.073 1.073 1.073Zm6.166-8.383-6.047.648c-.111.012-.168-.13-.08-.198l5.918-4.608c.385-.314.629-.803.524-1.327-.105-.804-.768-1.328-1.606-1.223l-6.431.941c-.113.017-.174-.128-.083-.197l6.374-4.867c1.258-.978 1.362-2.9.21-4.017-1.048-1.048-2.725-1.013-3.774.035L4.004 11.249c-.384.419-.558.978-.453 1.572.174.943 1.117 1.573 2.061 1.398l5.537-1.13c.12-.024.185.136.081.202l-6.143 3.932c-.769.489-1.117 1.362-.874 2.236.245 1.153 1.398 1.817 2.516 1.537l9.182-2.262c.103-.025.179.095.113.177l-1.435 1.77c-.384.49.245 1.153.769.769l4.717-3.877c.838-.699.279-2.062-.804-1.957l-.001.001Z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Envato</title><path d="M16.156 0a2.7 2.7 0 0 0-1.886.8L4 11.253c-.382.42-.558.978-.453 1.57.176.945 1.116 1.571 2.06 1.399l5.54-1.13c.12-.025.183.136.082.204L5.088 17.23c-.769.487-1.119 1.36-.875 2.234.244 1.151 1.398 1.814 2.516 1.537l9.183-2.26a.11.11 0 0 1 .113.176l-1.433 1.77c-.383.487.244 1.15.77.767l4.716-3.877c.84-.697.281-2.062-.803-1.957l-.011-.004-6.047.65a.111.111 0 0 1-.08-.199l5.918-4.609c.382-.315.628-.801.523-1.326-.105-.803-.767-1.328-1.607-1.223l-6.43.942c-.112.015-.174-.128-.084-.2l6.375-4.867c1.256-.978 1.36-2.898.209-4.015A2.6 2.6 0 0 0 16.156 0M13.1 21.855a1.07 1.07 0 0 0-1.073 1.073A1.07 1.07 0 0 0 13.1 24a1.07 1.07 0 0 0 1.072-1.072 1.07 1.07 0 0 0-1.072-1.073"/></svg>
```
Seeing some subtle differences. Remade it, optimized with precision 3 here. Does it look okay for you?

Reply:
Some minor differences

![envato](https://github.com/user-attachments/assets/a097e285-027e-4112-9405-2186341577d8)

Reply:
Not seeing the varying line widths on my end. Those lines in general are just how editors (my editor at least) shows them overlapping, even though the supposed differences shown by the lines aren't visible if you comb over the edges or subtract one path from the other (those are the two methods I use to compare).

Reply:
Outline view mode in Affinity Designer and Illustrator is useful for more precise analysis, but in my opinion, these minute differences are negligible.
![image](https://github.com/user-attachments/assets/85030d2e-9b1f-475c-81e2-3753152d1e52)

Reply:
I don't see much difference. At 64000% zoom, in outline view I see minor difference. That is fine.